### PR TITLE
Update document for shape estimation

### PR DESCRIPTION
**Pull request by wep21**
## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [pull request guidelines][pull-request-guidelines]
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: perception/object_recognition/detection/shape_estimation/README.md [hunk 1]
```diff
@@ -13,54 +19,28 @@ keywords = {autonomous driving, laser scanner, perception, segmentation},
...
 
-| Parameter | Type     | Description                                                             | Default                                     |
-| --------- | -------- | ----------------------------------------------------------------------- | ------------------------------------------- |
-| `input`   | _String_ | Topic name containing the objects detected by the Lidar in 3D space.    | `/detection/lidar_detector/objects`         |
-| `output`  | _String_ | Topic name containing the objects with the shape estimated in 3D space. | `/detection/lidar_shape_estimation/objects` |
+| Name                        | Type | Default Value | Description                                        |
+| --------------------------- | ---- | ------------- | -------------------------------------------------- |
+| `use_corrector`             | bool | true          | The flag to apply rule based filter.               |
+| `use_filter`                | bool | true          | The flag to apply rule based corrector             |
+| `use_vehicle_reference_yaw` | bool | true          | The flag to use vehicle reference yaw for corrector |
```
Comment (kenji-miyake) [inline perception/object_recognition/detection/shape_estimation/README.md:42]: typoを見つけてコメントしようと思ったら直りましたが、pre-commit落ちちゃいますね :pleading_face:

Comment (kenji-miyake) [inline perception/object_recognition/detection/shape_estimation/README.md:42]: - rule-basedに変更
- ピリオド削除
- スペース調整

```suggestion
| Name                        | Type | Default Value | Description                                        |
| --------------------------- | ---- | ------------- | -------------------------------------------------- |
| `use_corrector`             | bool | true          | The flag to apply rule-based filter                 |
| `use_filter`                | bool | true          | The flag to apply rule-based corrector              |
| `use_vehicle_reference_yaw` | bool | true          | The flag to use vehicle reference yaw for corrector |
```

**Comment by yukkysaito** [comment]
@wep21 L-shape fittingしかしてないみたいな感じに書かれてしまっているのでラベルに応じてBounding Box, cylinder, convexhullのfittingを行うという文に変えていただけますか？

**Comment by yukkysaito** [comment]
論文は
```
## (Optional) References/External links
```
を追加して移しておいていただけると助かります

**Comment by wep21** [comment]
> L-shape fittingしかしてないみたいな感じに書かれてしまっているのでラベルに応じてBounding Box, cylinder, convexhullのfittingを行うという文に変えていただけますか？

> 論文は
## (Optional) References/External links
を追加して移しておいていただけると助かります

I addressed your review at https://github.com/tier4/AutowareArchitectureProposal.iv/pull/604/commits/d826ddf644472b78a934b180a92adf197db46503.

**Review by yukkysaito — APPROVED** [review]

**Pull request merged**